.changeset/fuzzy-turtles-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+Moved analytics callback for expression widget to Renderer to hook on to prevent overfiring
```

Comment:
**todo**: Update this based on the latest round of changes.

Reply:
Ah, good eye! Thanks!